src/www/services_dhcpv6_edit.php

```diff
@@ -198,7 +198,7 @@ function staticmaps_sort($ifgui) {
 					                    <input name="duid" type="text" class="formfld unknown" id="duid" size="40" value="<?=htmlspecialchars($pconfig['duid']);?>" />
 					                    <br />
 					                    <span class="vexpl"><?=gettext("Enter a DUID Identifier in the following format: ");?><br />
-					"DUID-LLT - ETH -- TIME --- ---- address ----" <br />
+          "<?= gettext('DUID-LLT - ETH -- TIME --- ---- address ----') ?>" <br />
```

Comment:
Only adds to confusion, maybe change the string to "DUID-LLT - ETH -- TIME --- ---- ADDR ----" and leave it static?